web3swift/src/Account/EthereumAccount.swift

```diff
@@ -102,6 +102,10 @@ public class EthereumAccount: EthereumAccountProtocol {
         }
     }
     
+    public func sign(hash: Data) throws -> Data? {
```

Comment:
why not `public func sign(message: Data)` ?

Reply:
👍